src/main/java/servlets/VendorServlet.java

```diff
@@ -33,12 +45,73 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     JsonConverter converter = new JsonConverter();
     String configMap = converter.getConfigMap();
-    response.setContentType("text/html;");
+    response.setContentType(CONTENT_TYPE);
     response.getWriter().println(configMap);
   }
 
-  @Override
-  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // DELETE method --> deletes vendor configuration directory
+  /** Delete the desired Vendor from the fileset and spreadsheets.*/
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {  
+    String vendorID = request.getParameter(VENDOR_ID_PARAM);
+    String responseString = "success"; 
+
+    // Delete the file from the filesystem and update the sheets.
+    try {
+      deleteVendorFile(vendorID);
+      updateSheets(request);    
+    } catch (IOException e) {
+      responseString = "Failed to Delete File.";
+    } catch (GeneralSecurityException e) {
+      responseString = "Failed to Delete File.";
+    } 
```

Comment:
Since these do the same thing, we could use a single catch clause with:
`catch (IOException | GeneralSecurityException e)`

Reply:
Cool, I didn't know you could do that.